climakitae/core/data_load.py

```diff
@@ -547,7 +550,7 @@ def _merge_all(selections, data_dict):
         all_hist = xr.concat(
             [
                 _process_dset(one, data_dict[one], selections)
-                for one in data_dict.keys()
+                for one in tqdm(data_dict.keys(), "Processing datasets")
```

Comment:
"Retrieving datasets" sound good to me. It corresponds well with `retrieve()`. To avoid confusion, the message for `load` could emphasize on the process of bringing data into memory.

Reply:
`retrieve` can keep running after the progress bar indicates completion. The delay is especially noticeable when data size is large. Would it be feasible for the progress bar to monitor retrieve processes more completely? Otherwise the remaining processes may need to be communicated via message(s).